app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+# Create comment model
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] There is no column `author` in your database schema. The associations between models should be set up correctly according to your database schema.